data/map systems.txt

```diff
@@ -29501,7 +29557,6 @@ system Sheratan
 	fleet "Large Core Merchants" 1800
 	fleet "Small Core Pirates" 3000
 	fleet "Large Core Pirates" 5000
-	fleet "Korath Large Raid" 40000
```

Comment:
Ditto

Reply:
Korath raiding major Syndicate systems causes a lore problem: Korath cannot be secret if they're raiding populated Syndicate worlds. This story and its associated lore establishes the bounds of what they actually raid, and why. Any benefits Korath get from Syndicate space comes remotely through New Tortuga's illicit activities. (Such as jobs board jobs.)